src/components/app.js

```diff
@@ -1,36 +1,173 @@
-import icon from '../assets/menu.png';
-
 const addList = document.querySelector('.list-items');
+const btn = document.querySelector('.add-todo');
+const addElem = document.querySelector('.add-list');
+
+let lists;
+
+const saveItems = () => {
+  localStorage.setItem('data', JSON.stringify(lists));
+};
+
+const savedItems = JSON.parse(localStorage.getItem('data'));
+
+if (Array.isArray(savedItems)) {
+  lists = savedItems;
+} else {
+  lists = [];
+}
+
+class Node {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.index = `${lists.length}`;
```

Comment:
- [ ] Your first item in the storage gets the index equals to 0. However, it should have the length of the new array which is 1. Kindly fix this.